src/Getters/Holidays.php

```diff
@@ -30,8 +30,13 @@ public function holidaysHebrew()
      * @param  string  $format
      * @return array
      */
-    private function holidays($format)
+    private function holidays($format, $galus = null)
     {
+        if (!is_null($galus)) {
+            $current_galus = $this->galus;
+            $this->galus = $galus;
```

Comment:
The cleanest way to do this, would be to add a $galus flag to all holiday checks (`isChanukah($galus = null)`) even though it wouldn't mean anything, this way when we loop through all the holidays, we are properly calling those functions with a parameter, because only some of those functions expect a parameter

Reply:
What is this change intended to fix?

Reply:
we have already the setGalusMode though right? So why does holidays need to accept this toggle?